.gitignore

```diff
@@ -3,3 +3,5 @@ node_modules
 yarn.lock
 .mongo
 sms-utils/backups
+bundle.js
+bundle.js.map
```

Comment:
Nah, the production running docker container actually looks for the index.html file in dist. Not ideal, but the best I could do after 30 mins of dealing with webpack